tests/conftest.py

```diff
@@ -0,0 +1,3 @@
+import sys
+sys.path.append('../backend')
+sys.path.append('backend')
```

Comment:
Why two appends here ?

Reply:
These 2 lines tells Python runtime where to find the RPE Phyton classes which are located outside of the 'tests' folder. I put these 2 lines in here because the conftest.py is a pytest configuration which is automatically ran when pytest is executed.